recipe/deploy/shared.php

```diff
@@ -53,7 +62,15 @@
         // and file exist in release
         if (!test("[ -f $sharedPath/$file ]") && test("[ -f {{release_path}}/$file ]")) {
             // Copy file in shared dir if not present
-            run("cp -rv {{release_path}}/$file $sharedPath/$file");
+            run(
+                sprintf(
+                    'cp -r%s {{release_path}}/%s %s/%s',
+                    output()->getVerbosity() === OutputInterface::VERBOSITY_DEBUG ? 'v' : '',
+                    $file,
+                    $sharedPath,
+                    $file
+                )
```

Comment:
This look way too verbose. Better use $v var. Just inline it “-r$v”

Reply:
ok, it seemed to be more readable, but i guess different preferences 😄 changed it